src/subdomains/core/buy-crypto/route/buy.controller.ts

```diff
@@ -137,4 +139,19 @@ export class BuyController {
 
     return { ...Config.bank.dfxBankInfo, iban: bank.iban, bic: bank.bic };
   }
+
+  private getTransactionVolume(outputAsset: Asset): MinDeposit[] {
+    const blockchainVolumes = Object.entries(Config.blockchain.default.minTransactionVolume)
+      .filter(([key, _]) => key === outputAsset.blockchain)
+      .map(([_, value]) => value);
+
+    const minAssetVolume =
+      blockchainVolumes.length != 0
+        ? Object.entries(blockchainVolumes[0])
+            .filter(([key, _]) => key === outputAsset.name)
+            .map(([_, value]) => value)
+        : [];
+
+    return Util.transformToMinDeposit(minAssetVolume[0] ?? Config.blockchain.default.minTransactionVolume.default);
+  }
```

Comment:
Can we move the methods `getTransactionVolume` and `transformToMinDeposit` to the config class? Would make more sense there 🤔